torch_harmonics/distributed/distributed_convolution.py

```diff
@@ -257,23 +243,18 @@ def get_psi(self):
         psi = torch.sparse_coo_tensor(self.psi_idx, self.psi_vals, size=(self.kernel_size, self.nlat_out_local, self.nlat_in_local * self.nlon_in)).coalesce()
         return psi
 
-    def forward(self, x: torch.Tensor, use_triton_kernel: bool = True) -> torch.Tensor:
+    def forward(self, x: torch.Tensor, use_cuda_kernel: bool = True) -> torch.Tensor:
```

Comment:
I think this option should be removed. By default we'll use the cuda kernel if it is available

Reply:
fixed